selfdrive/car/toyota/toyotacan.py

```diff
@@ -33,18 +33,19 @@ def create_lta_steer_command(packer, steer_control_type, steer_angle, steer_req,
   return packer.make_can_msg("STEERING_LTA", 0, values)
 
 
-def create_accel_command(packer, accel, pcm_cancel, standstill_req, lead, acc_type, fcw_alert):
+def create_accel_command(packer, accel, accel_raw, pcm_cancel, standstill_req, lead, acc_type, fcw_alert):
   # TODO: find the exact canceling bit that does not create a chime
   values = {
     "ACCEL_CMD": accel,
     "ACC_TYPE": acc_type,
     "DISTANCE": 0,
     "MINI_CAR": lead,
-    "PERMIT_BRAKING": 1,
+    "PERMIT_BRAKING": 1 if accel < 0.3 else 0,
```

Comment:
I'd be careful with this, if you're on hills, the point at which you need to brake is much higher, so this adds opportunities for late braking. I tested messing with this signal, but could never get anything safe in all situations.

I recall Toyota keeping this bit at 1 behind leads, and acceleration feels fine there. It's possible that this is used for comfort when cruising without leads (to engine brake only)

Reply:
This is fine I think, I actually have this on my own fork for quite a while now, usually when going uphill the pcm will just stop accelerating, this should be okay, however if it's too low, say 0, it will create oscillations when going downhill which we dont want